db/migrate/20241021113348_make_fields_required_in_url_links.rb

```diff
@@ -0,0 +1,6 @@
+class MakeFieldsRequiredInUrlLinks < ActiveRecord::Migration[6.1]
```

Comment:
You could simplify your migration, and just rollback and update the original one, rather than having to have multiple migrations for this table. Might be good practice getting used to doing rollbacks and migrations?